### PR TITLE
SDK-3517 -  Support iOS SDK 6.0.0

### PR DESCRIPTION
**Pull request by AishwaryaNanna**
- Adds support for CT SDK v6.0.0 (thereby also v5.2.2)
(release for supporting ct-sdk v5.2.2 was skipped)
- Adds support for client-side in-apps.
- Adds new API `clearInAppResources(boolean)` to delete images and gifs which are preloaded for inapps in cs mode
- Adds new API `fetchInApps()` to explicitly fetch InApps from the server

### CODE REVIEW

**Review by akashvercetti — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 1.2.2 *()*
```
Comment (darshanclevertap) [inline CHANGELOG.md:4]: @akashvercetti @AishwaryaNanna Version is going to be 2.0.0

**Review by akashvercetti — APPROVED** [review]

**Review by CTLalit — COMMENTED** [review]
Branch name seems labelled wrongly.

**Comment by Anush-Shand** [comment]
@CTLalit @piyush-kukadiya please review the android related changes

**Review by darshanclevertap — APPROVED** [review]

**Review by Anush-Shand — APPROVED** [review]

**Comment by akashvercetti** [comment]
> Branch name seems labelled wrongly.

the branch was created for release with 5.2.2 a month ago. With 6.0.0 out, we have made the changes pointing to it.

**Pull request merged**